apps/e2e/src/trigger-client/fullSanity.ts

```diff
@@ -219,4 +219,5 @@ async function fullSanity(context: RunContext) {
   prettyPrint('List orders by digest result', ordersByDigest);
 }
 
-runWithContext(fullSanity);
+console.log('Running full sanity tests');
```

Comment:
nit: can we standardize log messages into some format where the client being tested (in this case, the `trigger-client`) is clear?